synapse/handlers/auth.py

```diff
@@ -46,6 +46,13 @@ def __init__(self, hs):
         }
         self.bcrypt_rounds = hs.config.bcrypt_rounds
         self.sessions = {}
+        self.INVALID_TOKEN_HTTP_STATUS = 401
+        self._KNOWN_CAVEAT_PREFIXES = set([
```

Comment:
I had pulled the stuff in synapse/api/auth.py down to here, but then the guest login stuff caused conflicts and I got lazy. Will fix.